metronome/examples/src/io/iohk/metronome/examples/robot/app/RobotComposition.scala

```diff
@@ -132,6 +133,8 @@ trait RobotComposition {
       viewStateStorage <- makeViewStateStorage(genesis)
       stateStorage     <- makeStateStorage(config, genesisState)
 
+      _ <- makeBlockPruner(config, blockStorage, viewStateStorage)
```

Comment:
Are you just tidying up or is there a functional reason to move it?

Reply:
The two composition classes were so similar in structure that I thought it will help anyone looking at both to see that the same thing appears in both at the same place, which is right after the storage components have been created. This should be less surprising than finding it further down.